project_event/models/project_task.py

```diff
@@ -224,6 +224,7 @@ class Task(models.Model):
         string='Actual End Time',
```

Comment:
Please dont do "git add . " You should always chose only the files that you modify, not all.

Reply:
@eilst  Ok